### PR TITLE
fix: monorepo and build deployments

### PR DESCRIPTION
**Pull request by varl**
Can be verified by looking at the deployed result from the status checks in this repo, e.g.

- Single package **without** build step: https://github.com/d2-ci/deploy-build-test-repo/tree/fix-deploys
- Single package **with** build step: https://github.com/d2-ci/deploy-build-test-repo-build/tree/fix-deploys
- Multiple packages **without** build step:
  - https://github.com/d2-ci/deploy-build-test-monorepo-a/tree/fix-deploys
  - https://github.com/d2-ci/deploy-build-test-monorepo-b/tree/fix-deploys
- Multiple packages **with** build step:
  - https://github.com/d2-ci/deploy-build-test-monorepo-a/tree/fix-deploys
  - https://github.com/d2-ci/deploy-build-test-monorepo-b/tree/fix-deploys

The sources example packages can be found here: https://github.com/dhis2/deploy-build/tree/master/examples

The workflow for testing is here: https://github.com/dhis2/deploy-build/blob/master/.github/workflows/deploy-build.yml

Would be nice to have automatic verification on the deployed results to d2-ci as a status check, but not implemented yet.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -5577,6 +5577,8 @@ __webpack_require__(661)
 __webpack_require__(380)
 __webpack_require__(580)
 
+shell.config.verbose = true
```
Comment (varl) [inline dist/index.js:5580]: This is the compiled action, the source is the next file.

**Comment by varl** [comment]
@janhenrikoverland & @amcgee 

This should resolve the problems you were seeing in Data Visualizer with regard to this action not deploying the correct `build` folder in:

- https://github.com/dhis2/data-visualizer-app/tree/chore/workflows-32.x
- https://github.com/dhis2/data-visualizer-app/tree/chore/workflows-33.x
- https://github.com/dhis2/data-visualizer-app/tree/v34
- https://github.com/dhis2/data-visualizer-app/tree/master

**Comment by janhenrikoverland** [comment]
Awesome!

**Comment by janhenrikoverland** [comment]
> * https://github.com/dhis2/data-visualizer-app/tree/v34

It's `34.x` now :smiley:

**Comment by amcgee** [comment]
> @janhenrikoverland & @amcgee
> This should resolve the problems you were seeing in Data Visualizer with regard to this action not deploying the correct build folder in:

Amazing, thanks @varl!!

**Pull request merged**